### PR TITLE
Feature/extend accumulated charts screen

### PR DESCRIPTION
**Pull request by m1awicki**

### CODE REVIEW

File: app/src/main/java/com/mobilecoronatracker/utils/DateUtils.kt [hunk 1]
```diff
@@ -6,6 +6,12 @@ import java.util.TimeZone
 
 fun getTodayTimestamp(): Long = (System.currentTimeMillis() / MILLIS_PER_DAY) * MILLIS_PER_DAY
 fun getYesterdayTimestamp(): Long = getTodayTimestamp() - MILLIS_PER_DAY
+fun getTimestampForDaysBefore(daysCount: Int, referenceTimestamp: Long = getTodayTimestamp()): Long {
```
Comment (MDikkii) [inline app/src/main/java/com/mobilecoronatracker/utils/DateUtils.kt:11]: Why can't we use a simpler approach like for yesterday?

File: app/src/main/java/com/mobilecoronatracker/data/persistence/dao/AccumulatedDataDao.kt [hunk 2]
```diff
@@ -30,12 +30,14 @@ abstract class AccumulatedDataDao : BaseDao<AccumulatedData> {
         SELECT * 
         FROM accumulated_data 
         WHERE entry_date>=:timestampStart AND entry_date<=:timestampEnd
+        ORDER By entry_date
```
Comment (MDikkii) [inline app/src/main/java/com/mobilecoronatracker/data/persistence/dao/AccumulatedDataDao.kt]: Minor: `By` should be `BY` to make consistent use of the SQL statements as UPPERCASE words

Comment (m1awicki) [reply app/src/main/java/com/mobilecoronatracker/data/persistence/dao/AccumulatedDataDao.kt]: fixed

File: app/src/main/java/com/mobilecoronatracker/utils/DateUtils.kt (hunk 1 above)
Comment (m1awicki) [reply app/src/main/java/com/mobilecoronatracker/utils/DateUtils.kt:11]: Definitely yes.

**Review by MDikkii — APPROVED** [review]

**Pull request merged**